HicServices/RDMP
# Task/rdmp 151 Custom Load Metadata Locations

Allows you to specify the location for each directory used by Data Loads.
The idea behind this is to allow DLS to store archives on a secondary disk due to storage concerns

To Test:
Create a data load
When setting the LocationOfFlatFiles, choose the advanced option and select some folders to use
Run the load and the folders should be populated with the data load files
